### DISCUSSION TITLE
Website blocked due to phishing?

### DISCUSSION THREAD

**Discussion by BillyCottrell** [microsoft/AdaptiveCards#8042] [Q&A]
Hi,

What is going on with the website I am trying to go to https://adaptivecards.io, but I am getting antivirus popups saying they blocked the connection because of phishing???

I don't know what is going on, but it's the first time trying to get on the website.

Any ideas?

Thanks in advance!

**Comment by almedina-ms** [comment]
Hi @BillyCottrell 

We are not facing this issue when opening with edge, can we know what browser and operative system you are using to open the website? it would also be of great help to know if you are opening the website from a search engine and a screenshot of the popup would also be helpful

**Reply by BillyCottrell** [reply]
Hi, my apologies for the late answer!

I am using a Windows 11 laptop, when opening adaptivecards.io in any browser (Firefox, Chrome or even Edge) by antivirus goes off.
Both search engine and direct URL will give this result.

Antivirus:
![afbeelding](https://user-images.githubusercontent.com/32330318/201641390-9c181e64-81ae-4a89-9592-71cfafc7186d.png)
Edge:
![afbeelding](https://user-images.githubusercontent.com/32330318/201641441-b6e74316-bd26-4e0a-a9c5-c9132f269b2d.png)
Firefox:
![afbeelding](https://user-images.githubusercontent.com/32330318/201641475-30d5c3f9-af0b-4472-a941-45e2c91a9f98.png)
Chrome:
![afbeelding](https://user-images.githubusercontent.com/32330318/201641514-c84a7d9f-e1ce-4d5b-90ae-c27f65d2d26d.png)

Any ideas?

Thanks in advance!

**Reply by almedina-ms** [reply]
Thank you for letting us know, I've contacted Avast in order to figure out why is adaptivecards.io being flagged for phishing

**Reply by fowl2** [reply]
It was also marked as Phishing by Zscaler - so I'm assuming it ended up on some list shared by vendors.

I raised Zscaler case 03741456 which they resolved, but didn't include any information on why.

**Reply by BillyCottrell** [reply]
Seems like this problem has been resolved now, as I am able to access it without any phishing notifications from Avast.

**Comment by BillyCottrell** [comment]
@almedina-ms it seems the problem has returned... Any ideas? Thanks!

**Reply by BillyCottrell** [reply]
I have notified avast about this false positive, and it has been removed now. Although it is possible that the issue might come back in the future.
